docs/docs/hydroflow_plus/quickstart/first-dataflow.mdx

```diff
@@ -25,22 +25,37 @@ cargo generate gh:hydro-project/hydroflow template/hydroflow_plus
 
 ## Writing a Dataflow
 
-In Hydroflow+, streams are attached to a **`Location`**, which is a virtual handle to a **single machine** (the `Process` type) or **set of machines** (the `Cluster` type). To write distributed programs, a single piece of code can use multiple locations.
+In Hydroflow+, streams are attached to a **location**, which is either a virtual handle to a **single machine** (the **`Process`** type) or **set of machines** (the **`Cluster`** type). To write distributed programs, a single piece of code can use multiple locations.
```

Comment:
"To write distributed programs, a single piece of code can use multiple locations" --> "A single piece of Hydroflow+ code can describe a distributed program that runs across multiple processes, each with its own local state (data)."